miden/tests/integration/stdlib/crypto/fri.rs

```diff
@@ -0,0 +1,6613 @@
+use super::build_test;
+
+#[test]
+fn fri_full_test() {
+    let source = "
+        use.std::crypto::fri
+        begin
+            exec.fri::verify_fri
+        end
+        ";
+
+    let tape = TAPE.to_vec();
+    
+    let test = build_test!(source, &[], &tape, vec![]);
+    test.expect_stack(&[]);
+}
+
+static TAPE: [u64; 6594] = [
```

Comment:
Could we put these into a separate file which we'd read from?

Reply:
Done!